docs/features/editorplaceholder/README.md

```diff
@@ -0,0 +1,25 @@
+---
+category: user-interface
+order: 60
+url: features/editorplaceholder
+menu-title: Editor Placeholder
+meta-title-short: Editor Placeholder
+---
+<!--
+Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Using Editor Placeholder
+
+<info-box info="">
+	This feature was introduced in CKEditor 4.15 and is provided through the <a href="https://ckeditor.com/cke4/addon/editorplaceholder">Editor Placeholder</a> plugin that is included in the Standard and Full presets available from the official CKEditor <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site. You can also {@link guide/dev/plugins/README add it to your custom build} with <a href="https://ckeditor.com/cke4/builder">online builder</a>.
```

Comment:
Are capitalized names still a thing? I thought we were not using it unless this is a specifically named feature.

Reply:
Yes, we should be consistent with rest of docs.

Reply:
It seems that rest of docs still use capitalized names for both plugins and Download site name :thinking:. So to be consistent we should also keep it here.